content/en/blog/_posts/2023-MM-DD-kubelet-podresources-api-ga.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: blog
+title: 'Kubernetes v1.28: Kubelet podresources API GA'
+date: 2023-MM-DD
+slug: kubelet-podresources-api-GA
+---
+
+**Author:**
+Francesco Romani (Red Hat)
+
+The podresources API is an API served by the kubelet locally on the node, which exposes the compute resources exclusively
```

Comment:
Could link to https://kubernetes.io/docs/concepts/architecture/nodes/ here. I recommend doing that.

Reply:
Thanks, fixed per your suggestion.